crates/bdk/src/descriptor/error.rs

```diff
@@ -42,6 +45,8 @@ pub enum Error {
     Miniscript(miniscript::Error),
     /// Hex decoding error
     Hex(bitcoin::hashes::hex::Error),
+    /// The provided wallet descriptors are not unique
+    NonUnique(ExtendedDescriptor),
```

Comment:
```suggestion
    ExternalAndInternalAreTheSame,
```
+more obvious name. I don't think we need the Extended descriptor in there. It doesn't help you debug anything really.